CMSgov/cms-carts-seds
# MDCT 371 - Update Footer

# Description

This "foot"-er is really going to "heel" our design woes. I'll be here all week.

Fixes [MDCT-371](https://qmacbis.atlassian.net/browse/MDCT-371)

Examples:

Desktop:
![Screen Shot 2022-06-13 at 1 14 31 PM](https://user-images.githubusercontent.com/19439679/173408526-19dd5772-e727-49ae-a3a8-1d80591f3ecf.png)

Tablet (760px, not 768px):
![Screen Shot 2022-06-13 at 1 15 11 PM](https://user-images.githubusercontent.com/19439679/173408629-84c73668-d729-4af7-afd6-100ae3ff6ef9.png)

Mobile:
![Screen Shot 2022-06-13 at 1 15 26 PM](https://user-images.githubusercontent.com/19439679/173408662-bc0bca6b-3592-44b8-a498-f3a63cb41974.png)


## How to test

- `./dev local`
- Scroll down and check out the footer
- Adjust screen size to make sure mobile is handled properly.

## Dependencies

Please spell out any dependencies for this change -- ex -- requires an install / update / migration etc

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
